homeassistant/components/neato/vacuum.py

```diff
@@ -24,9 +30,10 @@
 SCAN_INTERVAL = timedelta(minutes=5)
 
 SUPPORT_NEATO = SUPPORT_BATTERY | SUPPORT_PAUSE | SUPPORT_RETURN_HOME | \
-                 SUPPORT_STOP | SUPPORT_START | SUPPORT_CLEAN_SPOT | \
-                 SUPPORT_STATE | SUPPORT_MAP | SUPPORT_LOCATE
+    SUPPORT_STOP | SUPPORT_START | SUPPORT_CLEAN_SPOT | \
+    SUPPORT_STATE | SUPPORT_MAP | SUPPORT_LOCATE
 
+DOMAIN = 'neato'
```

Comment:
We should import this from the neato component as another name, eg `NEATO_DOMAIN`.